descriptors/ModuleDescriptor-template.json

```diff
@@ -1564,7 +1578,11 @@
         "overdue-fines-policies.collection.get",
         "lost-item-fees-policies.item.get",
         "lost-item-fees-policies.collection.get",
-        "pubsub.publish.post"
+        "pubsub.publish.post",
+        "checkout-lock-storage.checkout-locks.item.post",
+        "checkout-lock-storage.checkout-locks.item.delete",
+        "mod-settings.entries.collection.get",
```

Comment:
There's an unspoken rule about storage permissions - if we're adding an "item" permission we should also add "collection" one and vice versa. In other words, we're either adding both of them or none.

Reply:
Added mod-settings.entries.item.get permission.